src/components/projects/projectId/labeling/sessionId/main-component/NavigationBarTop.tsx

```diff
@@ -28,12 +27,10 @@ export default function NavigationBarTop(props: NavigationBarTopProps) {
     const displayId = useSelector(selectUserDisplayId);
     const userDisplayRole = useSelector(selectDisplayUserRole);
 
-    const [refetchAvailableLinks] = useLazyQuery(AVAILABLE_LABELING_LINKS, { fetchPolicy: 'no-cache' });
-
     useEffect(() => {
         if (userDisplayRole?.role == UserRole.ENGINEER || !SessionManager.labelingLinkData) return;
         const heuristicId = SessionManager.labelingLinkData.linkType == LabelingLinkType.HEURISTIC ? SessionManager.labelingLinkData.huddleId : null;
-        refetchAvailableLinks({ variables: { projectId: projectId, assumedRole: userDisplayRole?.role, assumedHeuristicId: heuristicId } }).then((result) => {
+        getAvailableLinks(projectId, user?.role, heuristicId, (result) => {
```

Comment:
not sure why this was changed but userDisplayRole?.role is not the same as user?.role, i think this is wrong